server/app/filters/SettingsFilter.java

```diff
@@ -26,11 +27,14 @@ public SettingsFilter(Provider<SettingsService> settingsService, Materializer ma
     this.materializer = checkNotNull(materializer);
   }
 
+  private static final ImmutableList<String> EXCLUDED_PATHS =
```

Comment:
**suggestion (non-blocking):** May be worth adding a comment above this explaining these paths don't use settings (similar to the text above the NonUserRoutePrefixes enum).